tests/api_test.py

```diff
@@ -297,7 +296,7 @@ def for_each_tripple_func(src: ScAddr, edge: ScAddr, trg: ScAddr):
         templ.triple("_class_node", sc_types.EDGE_ACCESS_VAR_POS_TEMP, ScAddr(0))
         templ.triple("_class_node", ScAddr(0), sc_types.NODE_VAR)
         templ.triple(ScAddr(0), ScAddr(0), sc_types.NODE_VAR)
-        search_result_list = client.template_search(templ)
+        search_result_list = client.template_search(templ, {})
```

Comment:
So, can you extend tests for TestClientTemplate? As I understand, it will be a good idea to divide tests for processed identifier, ScAddr object, template identifier and SCs example. 
Will client works correctly with sc-machine 0.6.0, 0.6.1, 0.7.0 after changing the interface? Where should we mark supported versions of sc-machine? I think about readme

Reply:
It is difficult to add tests for identifier, ScAddr object, template identifier and SCs example. There is needed tests with sc-server in ci.